spec/factories.rb

```diff
@@ -15,14 +15,16 @@
 
   factory :review do
     sequence(:title) { |n| "#{n} This is my pizza slice" }
-    sequence(:body) { |n| "#{n} Literally stumbled upon this place. I had been hearing 
-      recommendations about Regina for months, and the one time I get lost in Boston 
-      dumps me right on their doorstep. You can't ignore fate, even when it's 
+    sequence(:body) { |n| "#{n} Literally stumbled upon this place. I had been hearing
```

Comment:
Line is too long. [86/80]<br>Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.